scripts/libxml2/build.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+ADDITIONAL_FLAGS=
+
+./autogen.sh \
+    --prefix=${INSTALL_DIR} \
+    --host=${TARGET} \
+    --with-sysroot=${SYSROOT_PATH} \
+    --disable-shared \
+    --disable-fast-install \
+    --enable-static \
+    --with-zlib \
+    --with-pic \
+    --without-python \
+    --without-debug \
+    --without-lzma \
+    CC=${FAM_CC} \
+    AR=${FAM_AR} \
+    RANLIB=${FAM_RANLIB} \
+    ${ADDITIONAL_FLAGS} || exit 1
+
```

Comment:
Please remove the second empty line here, below and in the end of the download.sh file.